strings/snake_case_to_camel_pascal_case.py

```diff
@@ -26,14 +26,14 @@ def snake_to_camel_case(input: str, use_pascal: bool = False) -> str:
     ValueError: Expected boolean as use_pascal parameter, found <class 'str'>
     """
 
-    if not isinstance(input, str):
-        raise ValueError(f"Expected string as input, found {type(input)}")
+    if not isinstance(raw_input, str):
```

Comment:
```suggestion
    if not isinstance(input_, str):
```
Please do not use `raw_input` because that was a builtin in Python 2 which needs to remain dead.